ykrt/src/mt.rs

```diff
@@ -99,6 +48,24 @@ pub struct MT {
 }
 
 impl MT {
+    pub fn global() -> &'static Self {
```

Comment:
I don't understand what this is for.

It returns self in a really convoluted way?

Reply:
Notice: there is no `&self`!

Reply:
Is this a singleton in disguise?

Reply:
Yes.

Reply:
OK, I think I get it now. I've done this sort of thing with `lazy_static` before.

What's the advantage of `once_cell` over `lazy_static` or even `std::sync::Once`?

Reply:
Or `std::lazy::OnceCell` (same thing, just in std?)

Reply:
`sync::Once` is awkward to use; `once_cell` is a layer over `lazy_static`. I wouldn't be surprised if we change this particular aspect at some point, but I don't think it's a big deal either way right now.

Reply:
How about `std::lazy::OnceCell`? It's not stabilized, but would mean we don't have to pull any deps in.

Reply:
I'm not sure that an unstable API over a stable external package is a win TBH.

Reply:
OK, let's keep an eye on it in the future.

Reply:
Agreed.